src/runtime/agas/server/symbol_namespace_server.cpp

```diff
@@ -35,6 +35,15 @@
 #include <utility>
 #include <vector>
 
+#include <boost/regex.hpp>
+
+// Reuse functionaliy from perf-counters
+namespace hpx { namespace performance_counters { namespace detail
+{
+    HPX_EXPORT std::string regex_from_pattern(std::string const& pattern,
```

Comment:
Wouldn't it make sense to move this into `hpx::util`?

Reply:
Yah, may make sense to do that - but that's unrelated to the current PR.

Reply:
Someone once told me that nothing is more permanent than a temporary solution ;)

Reply:
I have not added any temporary solutions. I used what was there for years already.

Reply:
What I meant is the temporary solution of pulling in the performance counters here for a utility function that is in itself unrelated to both performance counters and symbol namespace.